src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenThatWeWantToGetDependenciesViaDesignTimeBuild.cs

```diff
@@ -1,399 +0,0 @@
-﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System.Collections.Generic;
-using Microsoft.Build.Framework;
-using Xunit;
-
-namespace Microsoft.NET.Build.Tasks.UnitTests
-{
-    public class GivenThatWeWantToGetDependenciesViaDesignTimeBuild
```

Comment:
We have moved this functionality to the `ResolvePackageAssets` task.  Can we add unit tests similar to these ones that are being deleted to the tests for `ResolvePackageAssets`?